src/openfhe.jl

```diff
@@ -381,3 +385,291 @@ function rotate(sv::SecureVector{<:OpenFHEBackend}, shift; wrap_by)
 
     secure_vector
 end
+
+
+############################################################################################
+# Matrix
+############################################################################################
+
+"""
+    init_matrix_rotation!(context::SecureContext{<:OpenFHEBackend}, private_key::PrivateKey,
+                          shifts, shape)
+
+Generate rotation keys for matrix rotation with `OpenFHE.EvalRotate` using the `private_key` and for the
+rotation indexes in `shifts`. `shifts` is a vector of pair values. The keys are stored in the given
+`context`. Positive shift defines rotation to the right/top, e.g. a rotation with a shift (1, 0):
+[1 2 3; 4 5 6; 7 8 9] -> [7 8 9; 1 2 3; 4 5 6].
+Negative shift defines rotation to the left, e.g. a rotation with a shift (0, 1):
+[1 2 3; 4 5 6; 7 8 9] -> [3 1 2; 6 4 5; 9 7 8].
+"""
+function init_matrix_rotation!(context::SecureContext{<:OpenFHEBackend}, private_key::PrivateKey,
+                               shifts, shape)
+    cc = get_crypto_context(context)
+    shifts_ = []
```

Comment:
```suggestion
    shifts_ = []
    nrows, ncols = shape
```
Maybe this makes it easier to understand the code below, if you work with `shift_row`/`shift_col` and `nrows`/`ncols`?